applications/RANSApplication/custom_elements/convection_diffusion_reaction_element_data/k_epsilon/epsilon_element_data.h

```diff
@@ -0,0 +1,107 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+#if !defined(KRATOS_K_EPSILON_HIGH_RE_ELEMENT_DATA_EPSILON_ELEMENT_DATA_H_INCLUDED)
+#define KRATOS_K_EPSILON_HIGH_RE_ELEMENT_DATA_EPSILON_ELEMENT_DATA_H_INCLUDED
+
+// System includes
+
+// Project includes
+#include "containers/variable.h"
+#include "geometries/geometry_data.h"
+#include "includes/node.h"
+#include "includes/process_info.h"
+#include "includes/ublas_interface.h"
+
+// Application includes
+#include "custom_elements/convection_diffusion_reaction_element_data.h"
+
+namespace Kratos
+{
+///@name  Functions
+///@{
+
+namespace KEpsilonElementData
```

Comment:
Data containers in the `FluidDynamicsApplication` are implemented as classes. Is there any technical limitation to follow the already existent design?